src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/InitialConsiderationService.java

```diff
@@ -60,7 +61,7 @@
 import static uk.gov.hmcts.ethos.replacement.docmosis.constants.InitialConsiderationConstants.VIDEO;
 import static uk.gov.hmcts.ethos.replacement.docmosis.constants.InitialConsiderationConstants.WITH_MEMBERS;
 import static uk.gov.hmcts.ethos.replacement.docmosis.constants.InitialConsiderationConstants.hearingTypeMappings;
-import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.Constants.MONTH_STRING_DATE_FORMAT;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.Constants.*;
```

Comment:
This will fail checkstyle, it might be worth checking your IDE settings as I've noticed your PRs tend to do this. Reach out if you need help setting it